chains/solana/contracts/tests/txsizing_test.go

```diff
@@ -45,49 +98,46 @@ func TestTransactionSizing(t *testing.T) {
-		"fqConfigPDA":           mustRandomPubkey(),
-		"fqLinkConfig":          mustRandomPubkey(),
+	routerTable := RouterTable{
+		RouterConfig:        mustRandomPubkey(),
+		SystemProgram:       solana.SystemProgramID,
+		BillingTokenProgram: solana.TokenProgramID,
+		RouterBillingSigner: mustRandomPubkey(),
+		SysVarInstruction:   solana.SysVarInstructionsPubkey,
+		FqBillingSinger:     mustRandomPubkey(),
+		FeeQuoterProgram:    mustRandomPubkey(),
+		FqConfigPDA:         mustRandomPubkey(),
+		FqLinkConfig:        mustRandomPubkey(),
+		RmnProgram:          config.RMNRemoteProgram,
+		RmnCurses:           config.RMNRemoteCursesPDA,
+		RmnConfig:           config.RMNRemoteConfigPDA,
 	}
 
-	offrampTable := map[string]solana.PublicKey{
-		"config":                mustRandomPubkey(),
-		"referenceAddresses":    mustRandomPubkey(),
-		"originChainConfig":     mustRandomPubkey(),
-		"sysVarInstruction":     solana.SysVarInstructionsPubkey,
-		"systemProgram":         solana.SystemProgramID,
-		"billingSinger":         mustRandomPubkey(),
-		"feeQuoterProgram":      mustRandomPubkey(),
-		"fqConfigPDA":           mustRandomPubkey(),
-		"billingTokenConfig":    mustRandomPubkey(),
-		"destChainConfig":       mustRandomPubkey(),
-		"arbMessagingSigner":    mustRandomPubkey(),
-		"tokenPoolSigner":       mustRandomPubkey(),
-		"offramp":               config.CcipOfframpProgram,
-		"fqAllowedPriceUpdater": mustRandomPubkey(),
+	offrampTable := OfframpTable{
+		Config:                mustRandomPubkey(),
+		ReferenceAddresses:    mustRandomPubkey(),
+		SysVarInstruction:     solana.SysVarInstructionsPubkey,
+		SystemProgram:         solana.SystemProgramID,
+		BillingSinger:         mustRandomPubkey(),
+		FeeQuoterProgram:      mustRandomPubkey(),
+		Offramp:               config.CcipOfframpProgram,
+		FqAllowedPriceUpdater: mustRandomPubkey(),
+		FqConfig:              mustRandomPubkey(),
+		RmnProgram:            config.RMNRemoteProgram,
+		RmnCurses:             config.RMNRemoteCursesPDA,
+		RmnConfig:             config.RMNRemoteConfigPDA,
 	}
 
-	tokenTable := map[string]solana.PublicKey{
-		"tokenAdminRegistryPDA": mustRandomPubkey(),
-		"poolLookupTable":       mustRandomPubkey(),
-		"poolProgram":           config.CcipTokenPoolProgram,
-		"poolConfig":            mustRandomPubkey(),
-		"poolTokenAccount":      mustRandomPubkey(),
-		"poolSigner":            mustRandomPubkey(),
-		"tokenProgram":          config.Token2022Program,
-		"mint":                  mustRandomPubkey(),
+	tokenTable := TokenTable{
+		TokenAdminRegistryPDA: mustRandomPubkey(),
+		PoolLookupTable:       mustRandomPubkey(),
+		PoolProgram:           config.CcipTokenPoolProgram,
+		PoolConfig:            mustRandomPubkey(),
+		PoolTokenAccount:      mustRandomPubkey(),
+		PoolSigner:            mustRandomPubkey(),
+		TokenProgram:          config.Token2022Program,
+		Mint:                  mustRandomPubkey(),
+		RouterSigner:          mustRandomPubkey(),
```

Comment:
@agusaldasoro please review that this list still makes sense for every program. I found inconsistencies with this test and the accounts that were actually static enough (meaning they didn't depend on the caller/chain/feetoken) to have in an ALT

Reply:
This test could be outdated as it only builds the txs, and it doesnt' execute anything